### PR TITLE
Fix CIFAR normalization mean values

### PR DESCRIPTION
**Pull request by mark-arm**
Original code accidentally used the standard deviation as the mean for CIFAR normalization. These values calculated from the torchvision.datasets.CIFAR10 training data.

### CODE REVIEW

**Review by AidenDurrant — APPROVED** [review]
Thank you very much, I must have miss-pasted. Best,
Aiden

**Comment by mark-arm** [comment]
No problem, great work on the repo though, it's nice to see reproduction experiments included (and they showed the normalization didn't matter very much!). If you want others to build on this definitely add a LICENSE file of some kind!
